postcss.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    plugins: [
+        require("autoprefixer")({
+            browsers: ["last 2 version", "ie >= 9", "ios 8"]
```

Comment:
Een mooie vervanging hiervoor zou wel browserslist zijn. Dan is het altijd up to date. Voor nu wel ie11 opnemen lager dan dat ondersteunen we toch niet.

Reply:
Laten we hier een volgende keer nog eens naar kijken, in principe gebruik je toch nooit de build van maat.css dus is het ook niet zo'n issue dat hier vendor prefixes voor ie9 worden toevoegd.